lib/routes/jwks.js

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const config = require('../config');
+
+const KEYS = (function() {
+  var priv = config.get('openid.key');
+
+  function pub(key) {
```

Comment:
The functionality of this helper is so critical to security, I think it deserves a comment to say "Extract just the public information from the key" and to tell future refactorers to be super careful about it.